### DISCUSSION TITLE
Private Accounts enhancements

### DISCUSSION THREAD

**Discussion by FerasK0** [robertodoering/harpy#647] [Ideas]
I think private accounts need more support in terms of features, here is some stuff that could improve their experience:
"I mentioned these improvements before in #528 but I made this discussion to group them in one place. "

1- private account icon 🔒 
2- unfollow confirmation
3- make the retweet button on their tweet inactive rather than pushing an error message.
4- follow requests tab somewhere

that's all I can think of :)

**Comment by robertodoering** [comment]
Yes definitely, there is an issue for this: #264
